aptos-move/move-examples/token_objects/knight/sources/knight.move

```diff
@@ -0,0 +1,535 @@
+        let name = string::utf8(KNIGHT_COLLECTION_NAME);
+        let uri = string::utf8(KNIGHT_COLLECTION_URI);
+
+        // Creates the collection with unlimited supply and without establishing any royalty configuration.
+        collection::create_unlimited_collection(
+            creator,
+            description,
+            name,
+            option::none(),
+            uri,
+        );
+    }
+
+    /// Creates the food collection.
+    fun create_food_collection(creator: &signer) {
+        // Constructs the strings from the bytes.
+        let description = string::utf8(FOOD_COLLECTION_DESCRIPTION);
+        let name = string::utf8(FOOD_COLLECTION_NAME);
+        let uri = string::utf8(FOOD_COLLECTION_URI);
+
+        // Creates the collection with unlimited supply and without establishing any royalty configuration.
+        collection::create_unlimited_collection(
+            creator,
+            description,
+            name,
+            option::none(),
+            uri,
+        );
+    }
+
+    /// Creates the food token as fungible token.
+    fun create_food_token_as_fungible_token(
+        creator: &signer,
+        description: String,
+        name: String,
+        uri: String,
+        fungible_asset_name: String,
+        fungible_asset_symbol: String,
+        icon_uri: String,
+        project_uri: String,
+        restoration_value: u64,
+    ) {
+        // The collection name is used to locate the collection object and to create a new token object.
+        let collection = string::utf8(FOOD_COLLECTION_NAME);
+        // Creates the food token, and get the constructor ref of the token. The constructor ref
+        // is used to generate the refs of the token.
+        let constructor_ref = token::create_named_token(
+            creator,
+            collection,
+            description,
+            name,
+            option::none(),
+            uri,
+        );
+
+        // Generates the object signer and the refs. The object signer is used to publish a resource
+        // (e.g., RestorationValue) under the token object address. The refs are used to manage the token.
+        let object_signer = object::generate_signer(&constructor_ref);
+        let property_mutator_ref = property_map::generate_mutator_ref(&constructor_ref);
+
+        // Initializes the value with the given value in food.
+        move_to(&object_signer, RestorationValue { value: restoration_value });
+
+        // Initialize the property map.
+        //let properties = property_map::prepare_input(vector[], vector[], vector[]);
+        let properties = property_map::prepare_input(
+            vector[string::utf8(b"Restoration Value")],
+            vector[string::utf8(b"0x1::string::String")],
+            vector[bcs::to_bytes(&string_utils::to_string(&restoration_value))]
+        );
+        property_map::init(&constructor_ref, properties);
+        // property_map::add_typed(
```

Comment:
Can you just delete this instead of leaving it commented out, which can be confusing to readers?